tests/test_partitioning.py

```diff
@@ -26,12 +26,12 @@ def test_simple(self):
         # part_coef_calc(inp2,StartP,EndP,deltaP)
 
         gt = lambda p: burnman.geotherm.anderson(p)
-        pressure = StartP
-        temperature = gt([StartP])[0]
+        pressure = StartP *1.e9
+        temperature = 2000
         (a, b) = burnman.calculate_partition_coefficient(
             pressure, temperature, rel_mol_per, 0.5)
-        self.assertFloatEqual(a, 0.0319126180713)
-        self.assertFloatEqual(b, 0.125395826369)
+        self.assertFloatEqual(a, 0.18453519778)
+        self.assertFloatEqual(b, 0.102938439776)
```

Comment:
There's a lot of superfluous code here. Suggest shortening. 
More generally, why the change in values? The bulk composition hasn't changed, and there isn't enough freedom in the system to allow changes in per:bdg ratios.

[The functions in partitioning.py should be replaced with some generalized linear algebra; at the moment it is really clunky. I'll open an issue.]